### PR TITLE
prefilter/multibuf: test with multiple packets

### PR DESCRIPTION
**Pull request by catenacyber**
Ticket: 7326

## Ticket

Redmine ticket: https://redmine.openinfosecfoundation.org/issues/7326

### CODE REVIEW

File: tests/prefilter-multibuf-multipkts/test.yaml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+requires:
+  min-version: 8
+
+args:
+- -k none
+
+checks:
+- filter:
```
Comment (jufajardini) [inline tests/prefilter-multibuf-multipkts/test.yaml:8]: Should we include a check that `sid: 1` should not match?

Comment (catenacyber) [reply tests/prefilter-multibuf-multipkts/test.yaml:8]: sid 1 was a leftover from #2093 where I found this bug

**Review by victorjulien — APPROVED** [review]

**Comment by victorjulien** [comment]
Not sure why, but this fails on my openbsd runner.

**Comment by catenacyber** [comment]
Force-pushed the pcap resaved with my Wireshark

Before :
input.pcap:  pcap capture file, microsecond ts, extensions (little-endian) - version 2.4 (No link-layer encapsulation, capture length 262144)
After:
input.pcap: pcap capture file, microsecond ts (little-endian) - version 2.4 (No link-layer encapsulation, capture length 262144)

**Comment by victorjulien** [comment]
Merged in #2118, thanks!

**Pull request closed without merging**